testing/trino-testing/src/main/java/io/trino/testing/sql/TestTable.java

```diff
@@ -27,6 +27,9 @@
 import static java.lang.String.format;
 import static java.lang.String.join;
 
+/**
+ * Consider using {@link RemoteObjectFactory} instead.
```

Comment:
The first line of a javadoc should explain what the think is. Did you mean `@see`?